third_party/VCL_v2/instrset.h

```diff
@@ -23,6 +23,8 @@
 #ifndef INSTRSET_H
 #define INSTRSET_H 20200
 
+#include "config.hh"

```

Comment:
Make wrapper function  in bparser/include, introduce interset.cc and interset.hh, add interset.cc into library (in CMAKE).